### DISCUSSION TITLE
Connecting Daly BMS

### DISCUSSION THREAD

**Discussion by Alestrix** [mr-manuel/venus-os_dbus-serialbattery#95] [General]
I've searched the Docs, Discussions, and Issues, but somehow can't seem to find any information on how to connect the Daly BMS to the Victron GX device.
Do I have to connect the CAN pins of the BMS to the RJ45 port of the MultiPlus as briefly discussed in [this](https://communityarchive.victronenergy.com/questions/142216/multiplus-ii-gx-with-daly-bms-and-can-bus-connecti.html) post?
Like this?
- BMS Pin 5 (Can_L) -> MP "BMS-Can" RJ45 Pin 8
- BMS Pin 4 (Can_H) -> MP "BMS-Can" RJ45 Pin 7
- BMS Pin 3 (GND) -> MP "BMS-Can" RJ45 Pin 3
- Set CAN Bus Profile to 500 kbps

Is this correct?
There's also discussion about some "Daly Interface Board (WNT)", some say they needed it, some say they got it working without, so the confusion gets even bigger...

**Comment by SamuelBrucksch** [comment]
My Daly has CAN onboard, so I guess you need to identify which Daly BMS Version you have. Some have it and some not, not sure how to identify though.

I think I used this to build my cable (Pinout in 3):
https://www.victronenergy.com/live/battery_compatibility:can-bus_bms-cable

**Reply by Alestrix** [reply]
Thanks! I think I do have the CAN version.
![Screenshot_20241026-090028](https://github.com/user-attachments/assets/92b964da-cc2e-4b12-88c1-e549dd43fbda)

I'm now also struggling with connecting the Daly PC Software (from inside a Windows VM since I completely moved to Linux) to the BMS so I can switch the protocol to Victron. The VM does see the USB-to-serial adapter, but there's no communication with the BMS.

Will try some more...

**Reply by rbeudel** [reply]
Why dont you use uart or RS485? It is running perfect.

**Reply by Alestrix** [reply]
@rbeudel I'd use anything as long as it works 😁 - do you have a link to a howto?

**Reply by rbeudel** [reply]
You have only to put the usb-to-serial adapter in the uart connector (the place for the bluetooth dongle) and the usb in the GX Device and the serial driver will connect. I think there is a Driver Problem with the Windows VM. You dont need to change the protocol of the BMS.

**Comment by mr-manuel** [comment]
@SamuelBrucksch could you please contact me on Discord? There are some improvements for the driver CAN ability that could also implemented for the Daly CAN.

**Comment by transistorgit** [comment]
CAN support for daly is already implemented
